nanofi/src/cxx/Plan.cpp

```diff
@@ -304,11 +304,11 @@ std::shared_ptr<minifi::Connection> ExecutionPlan::connectProcessors(std::shared
   connection->setSource(src_proc);
 
   utils::Identifier uuid_copy, uuid_copy_next;
-  src_proc->getUUID(uuid_copy);
+  uuid_copy = src_proc->getUUID();
   connection->setSourceUUID(uuid_copy);
```

Comment:
here, too, this could be `connection->setSourceUUID(src_proc->getUUID());` etc

Reply:
done